ftw/news/tests/test_news_listing_block.py

```diff
@@ -159,13 +159,21 @@ def test_news_listing_block_shows_lead_image(self, browser):
             'Textblock with image',
             browser.css(lead_image_css_selector).first.attrib['title']
         )
+        browser.login().visit(block, view='block_view')
```

Comment:
Shouldn't we be testing `news_listing` instead?

Reply:
ohh right 🙈